__tests__/app.test.js

```diff
@@ -77,4 +77,58 @@ describe('GET api/articles/:article_id', () => {
                 expect(body.msg).toBe("bad request")
             })
     })
+});
+
+describe('GET api/articles', () => {
+    test('sending an API request to api/articles returns a Status-200: responds with an object, with a key articles, which has the value of an array of ALL articles, in this case - 13 total.', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                const articles = body.articles
+                expect(articles).toHaveLength(13)
+            })
+    })
+    test('sending an API request to api/articles returns a Status-200: responds with an object, with a key articles, which has the value of an array of ALL articles, with the correct keys assigned', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                const articles = body.articles
+                articles.forEach((article) => {
+                    expect(article).toMatchObject({
+                        "author": expect.any(String),
+                        "title": expect.any(String),
+                        "article_id": expect.any(Number),
+                        "topic": expect.any(String),
+                        "created_at": expect.any(String),
+                        "votes": expect.any(Number),
+                        "article_img_url": expect.any(String),
+                        "comment_count": expect.any(String)
+                    })
+                })
+            })
+    })
+    test('sending an API request to api/articles returns a Status-200: responds with an object, with a key articles, which has the value of an array of ALL articles, with the correct keys assigned, and is sorted by created_at in descending order', () => {
```

Comment:
This isn't a great test as all the things inbetween could be in any order even though not likely.
Have a look at the package jest-sorted which provides a good way to check the ASC and DESC ordering of things.